core/src/main/java/com/pholser/junit/quickcheck/internal/generator/NullAllows.java

```diff
@@ -0,0 +1,21 @@
+package com.pholser.junit.quickcheck.internal.generator;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.*;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * <p>Mark a parameter of a {@link com.pholser.junit.quickcheck.Property}
+ * method with this annotation, in addition to {@link javax.annotation.Nullable},
+ * to configure the probability of generating a null value.</p>
+ */
+@Target({ PARAMETER, FIELD, ANNOTATION_TYPE, TYPE_USE })
+@Retention(RUNTIME)
```

Comment:
A couple of things here:

1) I think `NullsAllowed` or `NullAllowed` reads better for the name of this annotation.
2) What do you think about using only this annotation to mark nullable property parameters, instead of the others (`javax.annotation.Nullable`, etc.) in conjunction with this one?
3) 0.5 seems like a pretty high default. Could we make this 0.25, or 0.2, perhaps?